java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1551,13 +1552,14 @@ public Table aggregateWindowsOverTimeRanges(WindowAggregate... windowAggregates)
       for (Map.Entry<Integer, ColumnWindowOps> entry : groupedOps.entrySet()) {
         int columnIndex = entry.getKey();
         for (WindowAggregateOp operation : entry.getValue().operations()) {
-          aggColumnIndexes[opIndex] = columnIndex;
-          aggOperationIds[opIndex] = operation.getAggregateOp().nativeId;
-          aggPrecedingWindows[opIndex] = operation.getWindowOptions().getPreceding();
-          aggFollowingWindows[opIndex] = operation.getWindowOptions().getFollowing();
-          aggMinPeriods[opIndex] = operation.getWindowOptions().getMinPeriods();
+          aggColumnIndexes[opIndex]       = columnIndex;
+          aggOperationIds[opIndex]        = operation.getAggregateOp().nativeId;
+          aggPrecedingWindows[opIndex]    = operation.getWindowOptions().getPreceding();
+          aggFollowingWindows[opIndex]    = operation.getWindowOptions().getFollowing();
+          aggMinPeriods[opIndex]          = operation.getWindowOptions().getMinPeriods();
```

Comment:
nit: can we not change the witespace unless needed.

Reply:
Argh. This was the result of the `clang-format` changes introduced in #3259. The advice was to set up VSCode to automatically apply `clang-format` on save. I didn't realize that that would muck up the code this much.

Reply:
The changes of actual substance are here:
1. https://github.com/rapidsai/cudf/pull/4881/commits/a6d8957d33f6fe9cdb22b246240457ee0a246e88: JNI Support for `row_number`
1. https://github.com/rapidsai/cudf/pull/4881/commits/47d2551683aa80bca54efb1a1a74abcdb40698f4: (JNI) Support for descending timestamps
1. https://github.com/rapidsai/cudf/pull/4881/commits/61e5bfc6db793aa2051ff8a150bbca0d20f14fc8: Unspecified grouping keys

Reply:
clang-format shouldn't run on Java files, should it?

Reply:
Hmm... No, it shouldn't. I must've inadvertently enabled formatting for Java code as well, in VS Code. I should roll this back as well.